code/modules/mob/living/carbon/human/life.dm

```diff
@@ -364,9 +364,13 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 			if(prob(25))
 				slurring += 2
 			jitteriness = max(jitteriness - 3, 0)
+			if(has_trait(TRAIT_DRUNK_HEALING) && prob(50))
+				adjustBruteLoss(-0.25, 0)
+				adjustFireLoss(-0.175, 0)
```

Comment:
Remove the prob(50) and change the healing to `-0.125` and `-0.0875`, respectively